### DISCUSSION TITLE
有觉得震动不跟手吗？

### DISCUSSION THREAD

**Discussion by Attiv** [imfuxiao/Hamster#17] [Q&A]
开了震动觉得有点不跟手，甚至觉得有点卡

**Comment by imfuxiao** [comment]
> 开了震动觉得有点不跟手，甚至觉得有点卡

您好, 是1.1.0版本么?

**Reply by Attiv** [reply]
是的，今天早上更新的

**Reply by imfuxiao** [reply]
我去测试下, 还是你提前提供的输入方案吧

**Reply by Attiv** [reply]
是的，不是很卡，就是有一点点觉得不跟手，能感觉出来

**Reply by imfuxiao** [reply]
刚已调整震动触发点. 关联版本1.1.2

**Reply by Attiv** [reply]
这个提交审核了吗😂今天刷了一早上appstore 了

**Reply by imfuxiao** [reply]
昨晚已提交, 不过现在还在审核中, 估计要到明天了. 现在美国那边下班了.

**Reply by Attiv** [reply]
好的，我跟他们说一下，有考虑出个tf 的吗😂

**Reply by imfuxiao** [reply]
有上Testflight, 你要用测试版么?

**Reply by Attiv** [reply]
要要要

**Reply by imfuxiao** [reply]
需要你的app账户,  我邀请你.

**Reply by Attiv** [reply]
dml0dGExOTkxQGljbG91ZC5jb20=

**Reply by imfuxiao** [reply]
收到

**Reply by amorphobia** [reply]
收到了，谢谢

**Reply by imfuxiao** [reply]
收到

**Reply by Attiv** [reply]
有发吗？我邮箱还没收到[让我看看]

**Reply by imfuxiao** [reply]
外部测试还是需要苹果审核, 需要等等.

**Reply by wzxmer** [reply]
我也想要一个
d3p4bWVyQG91dGxvb2suY29t

**Reply by imfuxiao** [reply]
收到~

**Reply by Attiv** [reply]
刷了一天appstore 快魔怔了

**Reply by imfuxiao** [reply]
额~~~不用刷了, 美国那边还没上班呢...

**Reply by imfuxiao** [reply]
1.1.2已上架

**Reply by BTBMan** [reply]
@imfuxiao 您好，目前感觉还是有这个问题，声音和震动的体验不是很好，没有原生键盘来的舒服（震动貌似还好，声音的体验不是很好），有一点点延迟感，声音感觉有延迟并且比原生的输入法要小很多。（一个键或多个键连续快速按键的时候会感觉会很明显），作者可以试试和原生的对比一下。
